doc/adr/0002-use-component-level-micro-frontends.md

```diff
@@ -0,0 +1,29 @@
+# 2. Use component-level micro frontends
+
+Date: 2020-03-25
+
+## Status
+
+Proposed
+
+## Context
+
+Having [separate micro frontends for pages](0001-use-micro-frontends.md) requires duplication of the header and footer.
+
+We expect users will want to change the content of both, as well as their styling and behaviour.
+
+## Decision
+
+We will use [Edge Side Includes (ESIs)](https://www.w3.org/TR/esi-lang), limited to `<esi:include>`.
```

Comment:
Wonder if there's a potential use for `<esi:remove>` to insert some kind of defaults for components like an empty header?

Reply:
The ESI itself should be optional in the page's configuration. Can expand it in the future if there is a need.